docs/fundamentals/code-analysis/quality-rules/ca1716.md

```diff
@@ -46,17 +46,16 @@ Select a name that does not appear in the list of keywords.
 
 You can suppress a warning from this rule if you're convinced that the identifier won't confuse users of the API, and that the library is usable in all available languages in .NET.
 
-## Configurability
+## Configure code to analyze
 
-### Api surface
+You can configure these options for just this rule, for all rules, or for all rules in this category (Naming). For more information, see [Code quality rule configuration options](../code-quality-rule-options.md).
 
-You can configure which parts of your codebase to run this rule on, based on their accessibility. For example, to specify that the rule should run only against the non-public API surface, add the following key-value pair to an *.editorconfig* file in your project:
+Use the following options to configure which parts of your codebase to run this rule on.
 
-```ini
-dotnet_code_quality.ca1716.api_surface = private, internal
-```
+- [Include specific API surfaces](#include-specific-api-surfaces)
+- [Analyzed symbol kinds](#analyzed-symbol-kinds)
 
-You can configure this option for just this rule, for all rules, or for all rules in this category (Naming). For more information, see [Code quality rule configuration options](../code-quality-rule-options.md).
+[!INCLUDE[api-surface](~/includes/code-analysis/api-surface.md)]
 
 ### Analyzed symbol kinds
 
```

Comment:
Can you change line 62 to:

`You can configure the kinds of symbols that will be analyzed by this rule. The allowable values are:

- `Namespace`
- `NamedType`
- `Method`
- `Property`
- `Event`
- `Parameter`

Separate multiple values with a comma (`,`). The default value includes all of the symbol kinds in the previous list.